### DISCUSSION TITLE
Whether to support a set of json path syntax like mysql

### DISCUSSION THREAD

**Discussion by davedwwang** [apache/doris#24907] [Ideas]
When I developed the json function according to the mysql documentation, I found that mysql supports a complete set of json_path syntax, is it necessary for doris to support it？
https://github.com/mysql/mysql-server/blob/ea1efa9822d81044b726aab20c857d5e1b7e046a/sql-common/json_path.h#L322
/**
  A JSON path expression.

  From the user's point of view, a path expression is a string literal
  with the following structure. We parse this structure into a
  Json_path object:

      pathExpression ::= scope  pathLeg (pathLeg)*

      scope ::= dollarSign

      pathLeg ::= member | arrayLocation | doubleAsterisk

      member ::= period (keyName | asterisk)

      arrayLocation ::=
        leftBracket
          (arrayIndex | arrayRange | asterisk)
        rightBracket

      arrayIndex ::=
        non-negative-integer |
        last [ minus non-negative-integer ]

      arrayRange ::= arrayIndex to arrayIndex

      keyName ::= ECMAScript-identifier | ECMAScript-string-literal

      doubleAsterisk ::= **

      to ::= "to"

      last ::= "last"
*/

**Comment by LemonLiTree** [comment]
Can you give me an example of a json path that mysql supports but doris does not?
